src/gwc/src/main/java/org/geoserver/gwc/config/GWCConfig.java

```diff
@@ -28,6 +28,8 @@
 
     private boolean directWMSIntegrationEnabled;
 
+    private boolean requireTiledParameter = true;
```

Comment:
Wondering, is this object stored using XStream? I have vague memories that XStream dodges the normal construction mechanism and ends up ignoring field initializers too. I would give it a look.
Believe that normally we use Boolean object wrapper, and in the getter, if null, return the desired default value... at least, that's what we do for CatalogInfo objects :-D